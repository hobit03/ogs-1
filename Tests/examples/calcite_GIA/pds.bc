GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT 0.13229
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE PLY_BC_RIGHT
 $DIS_TYPE
  CONSTANT 0.0

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  H+
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT  1.e-07

#BOUNDARY_CONDITION
 $PCS_TYPE
 REACT_GIA
 $PRIMARY_VARIABLE 
  Ca2+
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT  1.0e-12

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Mg2+
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT 1.0e-03  

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Cl-
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT 2.0e-03  

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  CO32-
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT  4.6738e-12

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  HCO3-
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT  9.9954e-11

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Calcite
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT 3.9516e-12  

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Dolomite(dis) 
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT 1.1755e-12

#STOP

