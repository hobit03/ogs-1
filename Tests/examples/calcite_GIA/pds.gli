#POINTS
1 0.00 0.00 0.00        $NAME POINT1
2 0.50 0.00 0.00        $NAME POINT2
3 0.50 0.020 0.00       $NAME POINT3
4 0.0 0.020 0.00        $NAME POINT4
 $NAME
  PLY_Area
 $POINTS
  1
  2
  3
  4
  1
#POLYLINE
 $NAME
  PLY_BC_LEFT
 $POINTS
  1
  4
#POLYLINE
 $NAME
  PLY_BC_TOP
 $POINTS
  3
  4
#POLYLINE
 $NAME
  PLY_BC_RIGHT
 $POINTS
  2
  3
#POLYLINE
 $NAME
  PLY_BC_Bottom
 $POINTS
  1
  2
#STOP

