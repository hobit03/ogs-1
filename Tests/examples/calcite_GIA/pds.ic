GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.0

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  H+
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.257485511264e-010

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Ca2+
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.226714727745e-004

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Mg2+
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 2.010000000593e-010  //0.0

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Cl-
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.000000709057e-012  //0.0

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  CO32-
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  3.386323637667e-005

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  HCO3-
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  8.312424993181e-005

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Calcite
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 2.070001572722e-004  //about the value from Gems (in mol/kg water)

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Dolomite(dis) 
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.0e-10

#STOP

