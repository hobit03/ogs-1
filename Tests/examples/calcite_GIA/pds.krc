GeoSys Reactions
#KINREACTIONDATA      
 $SOLVER_TYPE
	1                ; Solver type
 $RELATIVE_ERROR	
	1.e-6		 ; rel. Error
 $MIN_TIMESTEP	
	1.e-6		; min Timestep
 $INITIAL_TIMESTEP	
	1.e-4       ; init Timestep
 $BACTERIACAPACITY
  100.0
 $NO_REACTIONS
   POLYLINE PLY_BC_LEFT
 $ACTIVITY_MODEL
    2 ; constant activity coefficient = 0, DH = 1, Davies = 2

#REACTION ; 
 $NAME
   R1 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   1 H+ + 1 CO32- = 1 HCO3- ; 
 $EQ_CONSTANT
 2.132615e+10           ; K values
 
#REACTION ; 
 $NAME
   R2 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   1 Ca2+ + 1 CO32- = 1 CaCO3 ; 
 $EQ_CONSTANT
 1675.377           ; K values
 
#REACTION ; 
 $NAME
   R3 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   1 Ca2+ + 1 H+ + 1 CO32- = 1 CaHCO3+ ; 
 $EQ_CONSTANT
 2.7202e+11           ; K values
 
#REACTION ; 
 $NAME
   R4 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   1 Mg2+ + 1 CO32- = 1 MgCO3 ; 
 $EQ_CONSTANT
 954.333          ; K values
 
#REACTION ; 
 $NAME
   R5 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   1 Mg2+ + 1 H+ + 1 CO32- = 1 MgHCO3+ ; 
 $EQ_CONSTANT
 2.4952e+11           ; K values

#REACTION ; 
 $NAME
   R2 ;
 $TYPE
   MIN_EQ_REACT;
 $EQUATION
   1 CO32- + 1 Ca2+ = 1 Calcite;
 $EQ_CONSTANT
   3.019256431e+8           ; K values

#REACTION ; 
 $NAME
   R3 ;
 $TYPE
   MIN_EQ_REACT;
 $EQUATION
   2 CO32- + 1 Ca2+ + 1 Mg2+ = 1 Dolomite(dis); 
 $EQ_CONSTANT
   3.46577209e+16          ; K values  
   
#STOP

