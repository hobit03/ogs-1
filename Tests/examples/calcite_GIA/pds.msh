#FEM_MSH
 $PCS_TYPE
  GROUNDWATER_FLOW
 $NODES
  144 
0 .374999999999669 0 0
1 .37083654835384 .01 0
2 .364583333332989 0 0
3 .36458333333355 .02 0
4 .375000000000173 .02 0
5 .360435956790066 .01 0
6 .1249999999997 0 0
7 .118749999457446 .0100000001240363 0
8 .114583333333058 0 0
9 .129745370280036 .0100000000206727 0
10 .11458333333381 .02 0
11 .12500000000052 .02 0
12 .0520833333332083 0 0
13 .0456814236111514 .0100347222222222 0
14 .0416666666665666 0 0
15 .0562246817130125 .010005787037037 0
16 .0416666666668402 .02 0
17 .0520833333335502 .02 0
18 .06249999999985 0 0
19 .0666624469522192 .0100009645061728 0
20 .0625000000002602 .02 0
21 .0729166666664916 0 0
22 .0770826300476502 .0100001607510288 0
23 .0729166666669702 .02 0
24 .0833333333331333 0 0
25 .0874998827857853 .0100000267918381 0
26 .0833333333336803 .02 0
27 .104166666666417 0 0
28 .0979166471310376 .0100000044653064 0
29 .0937499999997749 0 0
30 .108333330077476 .0100000007442177 0
31 .0937500000003903 .02 0
32 .1041666666671 .02 0
33 .0361508969907641 .00927662037037037 0
34 .031249999999925 0 0
35 .0312500000001301 .02 0
36 .135416666666351 0 0
37 .138888888888998 .01 0
38 .135416666667202 .02 0
39 .333333333332949 0 0
40 .328607253086408 .01 0
41 .32291666666627 0 0
42 .322916666667057 .02 0
43 .33333333333368 .02 0
44 .318865740740743 .01 0
45 .29166666666623 0 0
46 .287615740740789 .01 0
47 .28124999999955 0 0
48 .281250000000564 .02 0
49 .291666666667187 .02 0
50 .27777777777784 .01 0
51 .30208333333291 0 0
52 .297357253086453 .01 0
53 .30208333333381 .02 0
54 .354166666666309 0 0
55 .354166666666927 .02 0
56 .350115740740698 .01 0
57 .270833333332871 0 0
58 .270833333333941 .02 0
59 .26629602732853 .01 0
60 .31249999999959 0 0
61 .309027777777795 .01 0
62 .312500000000434 .02 0
63 .343749999999629 0 0
64 .340277777777749 .01 0
65 .343750000000304 .02 0
66 .260416666666191 0 0
67 .260416666667317 .02 0
68 .447916666666529 0 0
69 .444444444444409 .01 0
70 .437499999999834 0 0
71 .437500000000087 .02 0
72 .447916666666739 .02 0
73 .43275516546635 .01 0
74 .468749999999917 0 0
75 .46379987300667 .0107291666666667 0
76 .458333333333223 0 0
77 .458333333333391 .02 0
78 .468750000000043 .02 0
79 .454282407407379 .01 0
80 .395833333333057 0 0
81 .392361111111043 .01 0
82 .385416666666363 0 0
83 .385416666666826 .02 0
84 .395833333333478 .02 0
85 .380671832132986 .01 0
86 .42708333333314 0 0
87 .422919881687194 .01 0
88 .416666666666446 0 0
89 .416666666666782 .02 0
90 .427083333333435 .02 0
91 .412519290123401 .01 0
92 .406249999999751 0 0
93 .402199074074012 .01 0
94 .40625000000013 .02 0
95 .177083333332955 0 0
96 .171408822016568 .01 0
97 .166666666666304 0 0
98 .180555555555662 .01 0
99 .166666666667245 .02 0
100 .177083333333926 .02 0
101 .23958333333286 0 0
102 .234486599736083 .01 0
103 .229166666666209 0 0
104 .229166666667332 .02 0
105 .239583333334013 .02 0
106 .187499999999605 0 0
107 .191550925926032 .01 0
108 .187500000000607 .02 0
109 .145833333333002 0 0
110 .149884259259368 .01 0
111 .145833333333883 .02 0
112 .160397376543318 .01 0
113 .156249999999653 0 0
114 .156250000000564 .02 0
115 .197916666666256 0 0
116 .202064043209982 .01 0
117 .197916666667288 .02 0
118 .208333333332907 0 0
119 .213527664417388 .01 0
120 .208333333333969 .02 0
121 .218749999999558 0 0
122 .218750000000651 .02 0
123 .0257643711419949 .00940200617283951 0
124 .0208333333332833 0 0
125 .474223823704332 .0105989583333333 0
126 .479166666666696 .02 0
127 .0208333333334201 .02 0
128 .479166666666612 0 0
129 .24497183124402 .01 0
130 .255350198651044 .01 0
131 .485287303950716 .0117664930555556 0
132 .489583333333348 .02 0
133 .014710728523675 .0115670010288066 0
134 .0104166666666417 0 0
135 .223974599581203 .01 0
136 .249999999999511 0 0
137 .250000000000694 .02 0
138 .01041666666671 .02 0
139 .489583333333306 0 0
140 .5 .02 0
141 0 0 0
142 0 .02 0
143 .5 0 0
 $ELEMENTS
  188 
0 0 tri 0 1 2
1 0 tri 3 1 4
2 0 tri 3 5 1
3 0 tri 5 2 1
4 0 tri 6 7 8
5 0 tri 6 9 7
6 0 tri 10 7 11
7 0 tri 9 11 7
8 0 tri 12 13 14
9 0 tri 12 15 13
10 0 tri 16 13 17
11 0 tri 15 17 13
12 0 tri 18 15 12
13 0 tri 18 19 15
14 0 tri 17 15 20
15 0 tri 19 20 15
16 0 tri 21 19 18
17 0 tri 21 22 19
18 0 tri 20 19 23
19 0 tri 22 23 19
20 0 tri 24 22 21
21 0 tri 24 25 22
22 0 tri 23 22 26
23 0 tri 25 26 22
24 0 tri 27 28 29
25 0 tri 27 30 28
26 0 tri 31 28 32
27 0 tri 30 32 28
28 0 tri 8 30 27
29 0 tri 8 7 30
30 0 tri 32 30 10
31 0 tri 7 10 30
32 0 tri 14 33 34
33 0 tri 14 13 33
34 0 tri 35 33 16
35 0 tri 13 16 33
36 0 tri 29 25 24
37 0 tri 29 28 25
38 0 tri 26 25 31
39 0 tri 28 31 25
40 0 tri 36 9 6
41 0 tri 36 37 9
42 0 tri 11 9 38
43 0 tri 37 38 9
44 0 tri 39 40 41
45 0 tri 42 40 43
46 0 tri 42 44 40
47 0 tri 44 41 40
48 0 tri 45 46 47
49 0 tri 48 46 49
50 0 tri 48 50 46
51 0 tri 50 47 46
52 0 tri 51 52 45
53 0 tri 49 52 53
54 0 tri 49 46 52
55 0 tri 46 45 52
56 0 tri 2 5 54
57 0 tri 55 5 3
58 0 tri 55 56 5
59 0 tri 56 54 5
60 0 tri 47 50 57
61 0 tri 58 50 48
62 0 tri 58 59 50
63 0 tri 59 57 50
64 0 tri 60 61 51
65 0 tri 53 61 62
66 0 tri 53 52 61
67 0 tri 52 51 61
68 0 tri 63 64 39
69 0 tri 43 64 65
70 0 tri 43 40 64
71 0 tri 40 39 64
72 0 tri 41 44 60
73 0 tri 62 44 42
74 0 tri 62 61 44
75 0 tri 61 60 44
76 0 tri 54 56 63
77 0 tri 65 56 55
78 0 tri 65 64 56
79 0 tri 64 63 56
80 0 tri 57 59 66
81 0 tri 67 59 58
82 0 tri 68 69 70
83 0 tri 71 69 72
84 0 tri 71 73 69
85 0 tri 73 70 69
86 0 tri 74 75 76
87 0 tri 77 75 78
88 0 tri 77 79 75
89 0 tri 79 76 75
90 0 tri 80 81 82
91 0 tri 83 81 84
92 0 tri 83 85 81
93 0 tri 85 82 81
94 0 tri 86 87 88
95 0 tri 89 87 90
96 0 tri 89 91 87
97 0 tri 91 88 87
98 0 tri 92 93 80
99 0 tri 84 93 94
100 0 tri 84 81 93
101 0 tri 81 80 93
102 0 tri 88 91 92
103 0 tri 94 91 89
104 0 tri 94 93 91
105 0 tri 93 92 91
106 0 tri 82 85 0
107 0 tri 4 85 83
108 0 tri 4 1 85
109 0 tri 1 0 85
110 0 tri 70 73 86
111 0 tri 90 73 71
112 0 tri 90 87 73
113 0 tri 87 86 73
114 0 tri 76 79 68
115 0 tri 72 79 77
116 0 tri 72 69 79
117 0 tri 69 68 79
118 0 tri 95 96 97
119 0 tri 95 98 96
120 0 tri 99 96 100
121 0 tri 98 100 96
122 0 tri 101 102 103
123 0 tri 104 102 105
124 0 tri 106 98 95
125 0 tri 106 107 98
126 0 tri 100 98 108
127 0 tri 107 108 98
128 0 tri 109 37 36
129 0 tri 109 110 37
130 0 tri 38 37 111
131 0 tri 110 111 37
132 0 tri 97 112 113
133 0 tri 97 96 112
134 0 tri 114 112 99
135 0 tri 96 99 112
136 0 tri 113 110 109
137 0 tri 113 112 110
138 0 tri 111 110 114
139 0 tri 112 114 110
140 0 tri 115 107 106
141 0 tri 115 116 107
142 0 tri 108 107 117
143 0 tri 116 117 107
144 0 tri 118 116 115
145 0 tri 118 119 116
146 0 tri 117 116 120
147 0 tri 119 120 116
148 0 tri 121 119 118
149 0 tri 120 119 122
150 0 tri 34 123 124
151 0 tri 123 34 33
152 0 tri 78 125 126
153 0 tri 125 78 75
154 0 tri 33 35 123
155 0 tri 74 125 75
156 0 tri 127 123 35
157 0 tri 128 125 74
158 0 tri 129 102 101
159 0 tri 102 129 105
160 0 tri 59 130 66
161 0 tri 130 59 67
162 0 tri 131 132 126
163 0 tri 133 134 124
164 0 tri 135 102 104
165 0 tri 102 135 103
166 0 tri 125 131 126
167 0 tri 123 133 124
168 0 tri 135 121 103
169 0 tri 122 135 104
170 0 tri 136 129 101
171 0 tri 129 137 105
172 0 tri 130 136 66
173 0 tri 137 130 67
174 0 tri 125 128 131
175 0 tri 123 127 133
176 0 tri 127 138 133
177 0 tri 128 139 131
178 0 tri 121 135 119
179 0 tri 135 122 119
180 0 tri 129 136 130
181 0 tri 137 129 130
182 0 tri 132 131 140
183 0 tri 141 134 133
184 0 tri 141 138 142
185 0 tri 133 138 141
186 0 tri 140 139 143
187 0 tri 140 131 139
#STOP
