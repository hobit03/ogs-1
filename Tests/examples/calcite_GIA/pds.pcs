GeoSys-PCS: Processes ------------------------------------------------
#PROCESS
 $PCS_TYPE
 GROUNDWATER_FLOW
 $NUM_TYPE
   NEW   
 $ELEMENT_MATRIX_OUTPUT 
  0
#PROCESS
 $PCS_TYPE
  REACT_GIA
 $NUM_TYPE
   NEW   
#STOP

; Todo
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT

