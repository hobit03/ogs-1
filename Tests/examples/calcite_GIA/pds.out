GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  eta_0
  eta_1
  eta_2
  HEAD
  H+
  Ca2+
  Mg2+
  CO32-
  Cl-
  HCO3-
  Calcite
  Dolomite(dis)
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP
